ext/node/polyfills/testing.ts

```diff
@@ -123,6 +127,10 @@ class NodeTestContext {
             if (!newNodeTextContext[skippedSymbol]) {
               throw err;
             }
+          } finally {
+            try {
+              await after();
+            } catch { /* pass */ }
```

Comment:
Does Node actually ignore error from `after` hooks?

Reply:
Only when test is already failing. Fixed.

Reply:
Got it